20/javadoc/org/apache/kafka/server/quota/ClientQuotaCallback.html

```diff
@@ -0,0 +1,439 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
```

Comment:
The added javadoc seems right to me as https://issues.apache.org/jira/browse/KAFKA-6576 is supposed to be included in 2.0. cc @rajinisivaram